.github/workflows/continuous-integration.yml

```diff
@@ -29,3 +29,4 @@ jobs:
       - run: yarn install --frozen-lockfile
       - run: yarn build
       - run: yarn test
+        if: github.event_name == 'push_request'
```

Comment:
Nice job here, but don't you think that there is one better way of doing it?
Think from the perspective that we would have defined somewhere when to run these jobs and when to not.